src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -632,32 +651,25 @@ BigDecimal getMinimumVoteThreshold() {
         : null;
   }
 
-  // function: getMaxSkippedRanksAllowed
-  // purpose: getter for maxSkippedRanksAllowed rule
-  // returns: max skipped ranks allowed in this config (possibly null)
-  Integer getMaxSkippedRanksAllowed() {
-    return rawConfig.rules.maxSkippedRanksAllowed;
-  }
-
   // function: getUndeclaredWriteInLabel
   // purpose: getter for UWI label
   // returns: UWI label for this config
   String getUndeclaredWriteInLabel() {
-    return rawConfig.rules.undeclaredWriteInLabel;
+    return getString(rawConfig.rules.undeclaredWriteInLabel);
```

Comment:
The treatBlankAsUndeclaredWriteIn option doesn't seem to work, but I don't think it's your fault. I'll deal with that in a separate issue: #324